/*
 * This file was automatically generated by EvoSuite
 * Fri May 05 14:14:57 GMT 2023
 */

package com.example.task_evaluated;

import org.junit.Test;
import static org.junit.Assert.*;
import static org.evosuite.runtime.EvoAssertions.*;
import com.example.task_evaluated.task_14769;
import java.util.Vector;
import org.evosuite.runtime.EvoRunner;
import org.evosuite.runtime.EvoRunnerParameters;
import org.junit.runner.RunWith;

@RunWith(EvoRunner.class) @EvoRunnerParameters(mockJVMNonDeterminism = true, useVFS = true, useVNET = true, resetStaticState = true, separateClassLoader = true, useJEE = true) 
public class task_14769_ESTest extends task_14769_ESTest_scaffolding {

  @Test(timeout = 4000)
  public void test0()  throws Throwable  {
      task_14769 task_14769_0 = new task_14769();
      Vector<String> vector0 = new Vector<String>();
      // Undeclared exception!
      try { 
        task_14769_0.setSize(vector0, (-2953));
      } catch(ArrayIndexOutOfBoundsException e) {
         //
         // -2953
         //
         verifyException("java.util.Vector", e);
      }
  }

  @Test(timeout = 4000)
  public void test1()  throws Throwable  {
      task_14769 task_14769_0 = new task_14769();
      // Undeclared exception!
      try { 
        task_14769_0.setSize((Vector) null, 0);
      } catch(NullPointerException e) {
         //
         // no message in exception (getMessage() returned null)
         //
         verifyException("com.example.task_evaluated.task_14769", e);
      }
  }
}
